doctr/models/detection/differentiable_binarization/base.py

```diff
@@ -201,9 +201,10 @@ def compute_distance(
         square_dist_2 = np.square(xs - b[0]) + np.square(ys - b[1])
         square_dist = np.square(a[0] - b[0]) + np.square(a[1] - b[1])
         cosin = (square_dist - square_dist_1 - square_dist_2) / (2 * np.sqrt(square_dist_1 * square_dist_2) + eps)
+        cosin = np.clip(cosin, -1.0, 1.0)
```

Comment:
@odulcy-mindee @charlesmindee avoiding invalid values (This was the reason for the warnings)